### PR TITLE
Create 1.26 and 1.27 version streams for telegraf

### PR DESCRIPTION
**Pull request by mattmoor**
#### For new package PRs only
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

File: telegraf-1.26.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package:
+  name: telegraf-1.26
+  version: 1.26.3
+  epoch: 1
```
Comment (ajayk) [inline telegraf-1.26.yaml:4]:
```suggestion
  epoch: 0
```

**Review by ajayk — DISMISSED** [review]

File: telegraf-1.26.yaml (hunk 1 above)
Comment (ajayk) [inline telegraf-1.26.yaml:4]: Does the epoch needs to be set to `0` as this is a considered as a new package

Comment (mattmoor) [inline telegraf-1.26.yaml:4]:
```suggestion
  epoch: 0
```

Comment (mattmoor) [reply telegraf-1.26.yaml]: I think you are right as this is effectively a new package, but I used the original epoch out of an abundance of caution.  Changed to zero now.

**Review by ajayk — APPROVED** [review]

**Pull request merged**